core/domain/fs_domain_test.py

```diff
@@ -15,113 +15,245 @@
 # limitations under the License.
 
 """Tests for filesystem-related domain objects."""
+import logging
+import os
 
+from constants import constants
 from core.domain import fs_domain
+from core.platform import models
 from core.tests import test_utils
 import feconf
 
+app_identity_services = models.Registry.import_app_identity_services()
+(file_models,) = models.Registry.import_models([
+    models.NAMES.file
+])
+
 
 class ExplorationFileSystemUnitTests(test_utils.GenericTestBase):
     """Tests for the datastore-backed exploration file system."""
 
     def setUp(self):
         super(ExplorationFileSystemUnitTests, self).setUp()
-        self.user_id = 'abc@example.com'
+        self.user_email = 'abc@example.com'
+        self.fs = fs_domain.AbstractFileSystem(
+            fs_domain.ExplorationFileSystem('exploration/eid'))
 
     def test_get_and_save(self):
-        fs = fs_domain.AbstractFileSystem(
-            fs_domain.ExplorationFileSystem('exploration/eid'))
-        fs.commit(self.user_id, 'abc.png', 'file_contents')
-        self.assertEqual(fs.get('abc.png'), 'file_contents')
+        self.fs.commit(self.user_email, 'abc.png', 'file_contents')
```

Comment:
Hmmm.. I think the commit expects a user_id as you did earlier. You can refer https://github.com/oppia/oppia/pull/5972/files#diff-ceaab7e54d8ce7ddffb0f2c94f455c9eR107 to see how to register a user with the email, and retrieve the user's id.

Reply:
Done!